### DISCUSSION TITLE
How to start contributing?

### DISCUSSION THREAD

**Discussion by danishhansari** [appwrite/appwrite#4899] [Q&A] [answered]
Dear Maintainer,

I am writing to express my interest in getting involved in your community and contributing to your project. I am a passionate frontend developer with a strong interest in open source software and I believe that your project has the potential to make a real impact.

I would like to know more about how I can get involved and start contributing to your project. Are there any specific areas that you are currently looking for help in? Do you have any resources or guides that can help me get started as a contributor?

I am excited about the opportunity to work with your team and contribute to this important project. Please let me know if there is anything I can do to get started.

Thank you for considering my request.

**Comment by stnguyen90** [comment, marked as answer]
Welcome! 👋🏼 As with most projects, the best way to get started is to:

1. Look at the [contributing doc](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)
2. Look for issues labeled [`good first issue`](https://github.com/search?q=org%3Aappwrite+org%3Aopen-runtimes+org%3Autopia-php+label%3A%22good+first+issue%22++no%3Aassignee&type=issues&state=open)

There's also a detailed [video](https://youtu.be/aDYveq0wJFQ) on how to contribute to Appwrite.
